kujira/assetlist.json

```diff
@@ -2462,6 +2462,37 @@
           }
         }
       ]
+    },
+    {
+      "description": "The token of MCTest",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+          "exponent": 0
+        },
+        {
+          "denom": "mc-test",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+      "name": "MCTEST",
+      "display": "mctest",
+      "symbol": "MC01",
+      "coingecko_id": "mctest-protocol",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mctest.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mctest.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mctest.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/mctest.svg",
```

Comment:
Please remove the svg, it can only be included in the same image object if it looks visual identical, which it does not.